css_tasks/task11/index.html

```diff
@@ -0,0 +1,896 @@
+      </header>
+      <div class="clients__wrapper container">
+        <div class="clients__element">
+          <span><img src="assets/layer-47@2x.png" alt=""/></span>
+          <div class="clients-element__content">
+            <h4>Matthew Dix</h4>
+            <p class="clients-element__subtitle">Graphic Design</p>
+            <p class="clients-element__text">
+              Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do
+              eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut
+              enim ad minim.
+            </p>
+          </div>
+        </div>
+        <div class="clients__element">
+          <span><img src="assets/layer-49@2x.png" alt=""/></span>
+          <div class="clients-element__content">
+            <h4>Nick Karvounis</h4>
+            <p class="clients-element__subtitle">Graphic Design</p>
+            <p class="clients-element__text">
+              Sit amet, consectetur adipiscing elit, sed do eiusmod tempor
+              incididunt ut labore et dolore magna aliqua. Ut enim ad minim
+              veniam
+            </p>
+          </div>
+        </div>
+        <div class="clients__element">
+          <span><img src="assets/layer-50@2x.png" alt=""/></span>
+          <div class="clients-element__content">
+            <h4>Jaelynn Castillo</h4>
+            <p class="clients-element__subtitle">Graphic Design</p>
+            <p class="clients-element__text">
+              Ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod
+              tempor incididunt ut labore et dolore magna aliqua. Ut enim ad
+              minim veniam
+            </p>
+          </div>
+        </div>
+        <div class="clients__element">
+          <span><img src="assets/layer-51@2x.png" alt=""/></span>
+          <div class="clients-element__content">
+            <h4>Mike Petrucci</h4>
+            <p class="clients-element__subtitle">Graphic Design</p>
+            <p class="clients-element__text">
+              Dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor
+              incididunt ut labore et dolore magna aliqua. Ut enim ad minim.
+            </p>
+          </div>
+        </div>
+      </div>
+    </section>
+    <!-- BLOG -->
+    <section class="standard container">
+      <header class="main-header">
+        <h2 class="main-header__headline">Our stories</h2>
+        <p class="main-header__title">latest blog</p>
+        <div class="main-header__separator"></div>
+      </header>
+      <div class="standard__wrapper standard__wrapper--blog">
+        <div class="standard__element">
+          <div class="standard__image">
+            <picture>
+              <source
+                media="(max-width:1024px)"
+                srcset="assets/layer-54@2x.jpg"
+              />
+              <img src="assets/layer-54@3x.jpg" alt="Woman in the office" />
+            </picture>
+          </div>
+          <h4>Lorem ipsum dolor sit amet</h4>
```

Comment:
I would suggest that it should be link to blog post in this h tag

Reply:
Done